app/utils/client-hints.tsx

```diff
@@ -15,6 +15,14 @@ export const clientHints = {
 			return value === 'dark' ? 'dark' : 'light'
 		},
 	},
+	timeZone: {
+		cookieName: 'CH-time-zone',
+		getValueCode: `Intl.DateTimeFormat().resolvedOptions().timeZone`,
+		fallback: 'UTC',
+		transform(value: string | null) {
+			return value ?? 'UTC'
+		},
```

Comment:
In this case, we don't need to transform the value at all and the fallback is already defined. I think we should not require the `transform` function. So let's remove it here.

```suggestion
```

This change may necessitate a small update in the code below.